.pre-commit-config.yaml

```diff
@@ -40,7 +40,7 @@ repos:
     - id: trailing-whitespace
       exclude: "^r/.*?/_snaps/.*?.md$"
   - repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: "v14.0.6"
+    rev: "v18.1.1"
```

Comment:
This needs to be reverted, this will bump the required clang-format for everyone. If you don't have 14.0.6 locally you can download a pre-built copy from the LLVM homepage.